data/RPK/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Republik",
+  "symbol": "RPK",
+  "decimals": 18,
+  "description": "RPK is utility token for Republik platform.",
+  "website": "https://republik.gg",
+  "twitter": "@RepubliK_GG",
+  "tokens": {
+    "goerli": {
+      "address": "0xc483695B56e20EA80d33aF7EEfDec9F08c2837A7"
+    },
+    "mantle-goerli": {
+      "address": "0xc483695B56e20EA80d33aF7EEfDec9F08c2837A7"
```

Comment:
https://explorer.testnet.mantle.xyz/address/0xc483695B56e20EA80d33aF7EEfDec9F08c2837A7/contracts?contract-tab=code#address-tabs
![20240104-145234](https://github.com/mantlenetworkio/mantle-token-lists/assets/16064604/1fe97735-563d-43a6-88e0-90df404c8d6f)

During the initialization of the l2 contract, the parameters address _l2Bridge and address _l1Token were passed incorrectly.

Reply:
Redeploy to https://explorer.testnet.mantle.xyz/address/0xc251BA15a91C4aca99a21b76d50e9B793a838421/contracts#address-tabs